server/events/runtime/runtime.go

```diff
@@ -29,5 +30,6 @@ func GetPlanFilename(workspace string, maybeCfg *valid.Project) string {
 	if maybeCfg == nil || maybeCfg.Name == nil {
 		return fmt.Sprintf("%s.tfplan", workspace)
 	}
-	return fmt.Sprintf("%s-%s.tfplan", *maybeCfg.Name, workspace)
+	unesacpedFilename := fmt.Sprintf("%s-%s.tfplan", *maybeCfg.Name, workspace)
```

Comment:
There's probably more characters than just `/` that aren't allowed in a filename. Let's filter using [a-zA-Z0-9-_] and replace with `-`.

We should do this filtering for every file returned from this function so you'll need to also handle the case of `	if maybeCfg == nil || maybeCfg.Name == nil {`